litmus-portal/frontend/src/components/CustomResultModal/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from "react";
 import { useStyles } from "./styles";
-import { Typography } from "@material-ui/core";
+import { Typography, createMuiTheme } from "@material-ui/core";
```

Comment:
Why is createMuiTheme import required outside theme data file?

Reply:
- [x] I've removed  createMuiTheme import from this file and added the changes in the main theme data